docs/content/node-operation/observer-node.mdx

```diff
@@ -75,16 +75,16 @@ curl -sL -O storage.googleapis.com/flow-genesis-bootstrap/boot-tools-intel-mac.t
 The `root-protocol-state-snapshot.json` is generated for each [spork](https://developers.flow.com/nodes/node-operation/spork) and contains the genesis data for that spork.
 It is published and made available after each spork. The download location is specified [here](https://github.com/onflow/flow/blob/master/sporks.json) under [rootProtocolStateSnapshot](https://github.com/onflow/flow/blob/master/sporks.json#L16) and can be downloaded as follows,
 
-For mainnet-19
+For mainnet-20
```

Comment:
rather than updating these every spork, could you tweak the wording so the reader can just substitute in the correct spork number?

Reply:
made a change to not use hardcoded spork numbers but instead have a generic instruction to get the latest spork.